.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,102 @@
+          username: ${{ secrets.DEV_USERNAME }}
+          key: ${{ secrets.DEV_SSH_KEY }}
+          port: ${{ secrets.DEV_PORT }}
+          script: |
+            cd /opt/app
+            export DOCKERHUB_USERNAME="${{ secrets.DOCKERHUB_USERNAME }}"
+            export DOCKERHUB_TOKEN="${{ secrets.DOCKERHUB_TOKEN }}"
+            export IMAGE_TAG="${{ needs.build-and-push.outputs.tags }}"
+            cd ~/deploy
+            chmod +x ./deploy.sh
+            ./deploy.sh
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)